vlc-android/src/org/videolan/vlc/gui/video/VideoPlayerActivity.java

```diff
@@ -1685,7 +1687,8 @@ public void displayWarningToast() {
         warningToast.show();
     }
 
-    void setAudioVolume(int vol) {
```

Comment:
What's the point of removing this parameter?

It make code more confusing, and current state is obfuscated.